src/index.d.ts

```diff
@@ -2,5 +2,6 @@ import JSONRPCClient from './lib/json-rpc-client'
 declare global {
   interface Window {
     client: JSONRPCClient
+    vega: any
```

Comment:
chore (non-blocking): Can we have a ticket to type this? Out of scope for this PR because it will require a decent amount of work for different TX types, but not something we should leave as an any if we can eventually.